supervisor/resolution/module.py

```diff
@@ -167,10 +165,10 @@ async def load(self):
     async def healthcheck(self):
         """Scheduled task to check for known issues."""
         await self.check.check_system()
+        await self.evaluate.evaluate_system()
 
         # Run autofix if possible
-        if self.sys_core.state == CoreState.RUNNING:
-            await self.fixup.run_autofix()
+        await self.fixup.run_autofix()
```

Comment:
This will now cause an error during startup:

```
21-04-12 16:31:10 WARNING (MainThread) [supervisor.jobs] 'ResolutionFixup.run_autofix' blocked from execution, system is not running - CoreState.SETUP
```

This is not a problem, but users are most likely to create issues / ask questions about it.

Reply:
that is fix on the current dev